core/templates/dev/head/pages/exploration-editor-page/editor-tab/training-panel/training-panel.directive.ts

```diff
@@ -15,6 +15,9 @@
 /**
  * @fileoverview Directive for the training panel in the state editor.
  */
+require(
```

Comment:
Missing require for `angular-html-bind`.

Reply:
Added.